### PR TITLE
포스트 옵션 수정 gql

### PR DESCRIPTION
**Pull request by robin-maki**

### CODE REVIEW

**Comment by robin-maki** [comment]
Current dependencies on/for this PR:
* main
  * **PR #251** <a href="https://app.graphite.dev/github/pr/penxle/penxle/251" target="_blank">Graphite</a> 
    * **PR #252** <a href="https://app.graphite.dev/github/pr/penxle/penxle/252" target="_blank">Graphite</a> 
      * **PR #255** <a href="https://app.graphite.dev/github/pr/penxle/penxle/255" target="_blank">Graphite</a>  👈

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/penxle/penxle/255?utm_source=stack-comment).

File: apps/penxle.com/src/lib/server/graphql/schemas/post.ts [hunk 1]
```diff
@@ -144,6 +144,17 @@ const DeletePostInput = builder.inputType('DeletePostInput', {
   }),
 });
 
+const UpdatePostOptionsInput = builder.inputType('UpdatePostOptionsInput', {
+  fields: (t) => ({
+    postId: t.id(),
+    visibility: t.field({ type: PostVisibility }),
```
Comment (devunt) [inline apps/penxle.com/src/lib/server/graphql/schemas/post.ts:150]: P1: `visibility`도 `required: false` 적용 부탁드려용

**Review by devunt — CHANGES_REQUESTED** [review]

**Review by devunt — APPROVED** [review]

File: apps/penxle.com/src/lib/server/graphql/schemas/post.ts [hunk 2]
```diff
@@ -288,14 +293,14 @@ builder.mutationFields((t) => ({
...
-          option: { create: { id: createId(), ...options } },
+          option: { create: { id: createId(), ...defaultOptions } },
         },
         update: {
           space: { connect: { id: input.spaceId } },
           member: { connect: { id: meAsMember.id } },
           user: { connect: { id: context.session.userId } },
           revisions: { create: { id: createId(), ...revision } },
-          option: { update: { ...options } },
+          option: { update: { ...defaultOptions } },
```
Comment (robin-maki) [inline apps/penxle.com/src/lib/server/graphql/schemas/post.ts:303]: 포스트가 업데이트될때는 이거 수정하면 안되네요!!!!!

Comment (robin-maki) [reply apps/penxle.com/src/lib/server/graphql/schemas/post.ts:303]: 셀프로는 request change 못 하네요... 암튼 머지하지 말구 잠시만 기다려주세요...

**Comment by robin-maki** [comment]
### Merge activity

* **Oct 27, 2:30 AM**: @robin-maki started a stack merge that includes this pull request via [Graphite](https://app.graphite.dev/github/pr/penxle/penxle/255).

**Pull request merged**